tests/masternode/test_close.py

```diff
@@ -0,0 +1,32 @@
+"""
+Provide tests for command line interface's masternode close command.
+"""
+import json
+
+from click.testing import CliRunner
+
+from cli.constants import PASSED_EXIT_FROM_COMMAND_CODE
+from cli.entrypoint import cli
+
+
+def test_close_node(mocker, close_masternode_transaction):
+    """
+    Case: close the masternode.
+    Expect: close masternode transaction's batch identifier is returned.
+    """
+    mock_mock_get_node_private_key = mocker.patch('cli.config.NodePrivateKey.get')
+    mock_mock_get_node_private_key.return_value = '42dada12f863528bd456785d8c544154db6ec9455be2c123d91b687df3697314'
+
+    mock_close_masternode = mocker.patch('cli.node.service.loop.run_until_complete')
```

Comment:
How does the test work with this?
```suggestion
    mock_close_masternode = mocker.patch('cli.masternode.service.loop.run_until_complete')
```

Reply:
@anastasiia-bilova, done.